pkg/scaling/resolver/aws_secretManager_handler.go

```diff
@@ -0,0 +1,120 @@
+package resolver
```

Comment:
`aws_secretManager_handler` doesn't follow the naming convention, could you rename the file to something like `aws_secretmanager_handler` or `aws_secret_manager_handler` ?

Reply:
Alright